src/ui/common/src/components/workflows/WorkflowSettings.tsx

```diff
@@ -278,25 +335,45 @@ const WorkflowSettings: React.FC<WorkflowSettingsProps> = ({
 
   const handleDeleteWorkflow = (event) => {
     event.preventDefault();
+
     setIsDeleting(true);
 
+    const data = { force: true };
```

Comment:
what is this 'data' for?

Reply:
`data` is what is sent to the request. There are two items:
1. `external_delete`: the saved objects to remove
2. `force`: whether to force deletion even if some saved objects have append update modes.

`force` is toggled in the SDK with the force argument to the function but here, we present the integrations and update modes to the user and they enter the workflow name to show that they understand and agree the whole thing is deleted so `force=True` when deleting from the UI.

Reply:
Sounds good!